src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -106,6 +106,8 @@ public void buildGraph() {
          * Use map based on the list of edges, so that only distinct transfers are stored. */
         Map<TransferKey, PathTransfer> distinctTransfers = new HashMap<>();
         RegularStop stop = ts0.getStop();
+        var fromParentStation = stop.getParentStation();
+
```

Comment:
you can check the `isNoTransfers` on `fromParentStation` already here

Reply:
See below, use `stop.transfersNotAllowed()` here as well.

Reply:
Changed